lib/updateCMS.ts

```diff
@@ -0,0 +1,20 @@
+"use server";
+
+import { createDirectus, rest, updateItems, staticToken } from "@directus/sdk";
+
+export const updateCMS = async (ids: string[]) => {
+  const accessToken = process.env.NEXT_CMS_ACCESS_TOKEN as string;
```

Comment:
This is the 3rd, now it really should be a global exposed variable wrapped in assertExists